pwndbg/commands/heap.py

```diff
@@ -166,23 +122,112 @@ def malloc_chunk(addr):
     """
     Prints out the malloc_chunk at the specified address.
     """
+    main_heap = pwndbg.heap.get_heap()
+
     if not isinstance(addr, six.integer_types):
         addr = int(addr)
 
     chunk = value_from_type('struct malloc_chunk', addr)
     size = int(chunk['size'])
-    prev_inuse = (size & PREV_INUSE) == 1
-    is_mmaped = (size & IS_MMAPED) == 1
-    non_main_arena = (size & NON_MAIN_ARENA) == 1
+
+    prev_inuse, is_mmapped, non_main_arena = main_heap.chunk_flags(size)
 
     header = M.get(addr)
     if prev_inuse:
```

Comment:
Also, if there's any way to save a bunch of lines in the above screenshot by moving the opening brace onto the previous line, that'd be greeeeeeeaaaaaat </Lumberg>.